### PR TITLE
Advise reporter consumer

### PR DESCRIPTION
**Pull request by pacospace**
## This introduces a breaking change

- [ ] Yes
- [x] No

## This Pull Request implements

Advise Reporter consumer

### CODE REVIEW

File: process_message.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+import logging
+import os
+
+from thoth.common import init_logging, OpenShift
+from prometheus_client import CollectorRegistry, Gauge, Counter, Summary, push_to_gateway
+
+init_logging()
+
+_LOGGER = logging.getLogger("thoth.package_update")
```
Comment (KPostOffice) [inline process_message.py]: Copy pasta

File: process_message.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+REQUEST_TIME = Summary("request_processing_seconds", "Time spent processing request")
+
+
+@REQUEST_TIME.time()
+def process_advise_justification(advise_justification):
+    """Process a advise justification message from advise-reporter producer."""
+    _METRIC_ADVISE_TYPE.labels(advise_justification["message"]).set(advise_justification["count"])
+
+    if _THOTH_METRICS_PUSHGATEWAY_URL:
```
Comment (KPostOffice) [inline process_message.py]: Why are you using a push gateway can't we add a service if this is a deployment?

File: process_message.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
+#!/usr/bin/env python3
+# thoth-package-update
+# Copyright(C) 2020 Kevin Postlethwait
```
Comment (KPostOffice) [inline process_message.py]: I think you should get some credit too

File: process_message.py (hunk 1 above)
Comment (pacospace) [reply process_message.py]: :spaghetti: thanks

File: process_message.py (hunk 2 above)
Comment (pacospace) [reply process_message.py]: You are right! Thanks Kevin

**Review by KPostOffice — APPROVED** [review]
This looks good to go :+1:

**Pull request merged**